public/assets/custom-style.js

```diff
@@ -44,10 +44,20 @@ const observerMainApp = new MutationObserver((mutations) => {
       mutation.target.className == 'hide-for-sharing headerGlobalNav') {
       $(mutation.target).find('a[href$="app/kibana_overview"]').parent().addClass('hide');
     }
+    
+    let homeLink
     /**
      * Fix top-left Logo home link
      */
-    const homeLink = document.querySelector('#globalHeaderBars a.euiHeaderLogo[href$="/app/home"]');
+    homeLink = document.querySelector('#globalHeaderBars a.euiHeaderLogo[href$="/app/home"]');
+    if (homeLink) {
+      changeHomeLink(homeLink);
+    }
+
+    /**
+     * Fix navigation drawer Home link
+     */
+    homeLink = document.querySelector('nav a.euiListGroupItem__button[href$="/app/home"]');
```

Comment:
**suggestion**: I would prefer not to reuse the same variable to select different elements.

Reply:
Checked